src/components/Test.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { fb } from '../lib/firebase';
+
+const Test = () => {
+  const db = fb.firestore();
```

Comment:
If you wanted, you could also export this from [`lib/firebase`](https://github.com/the-collab-lab/tcl-17-smart-shopping-list/blob/c3508f7adf120921814dab7e4b38ef078234ee56/src/lib/firebase.js#L18), then just import it here. It might be useful going forward.